lib/features/chat/view/screens/pinned_messages_screen.dart

```diff
@@ -53,9 +53,9 @@ class _PinnedMessagesScreen extends ConsumerState<PinnedMessagesScreen>
   int _currentMatch = 1;
 
   // ignore: unused_field
-  int _currentMatchIndex = 0;
-  Map<int, List<MapEntry<int, int>>> _messageMatches = {};
-  List<int> _messageIndices = [];
+  final int _currentMatchIndex = 0;
+  final Map<int, List<MapEntry<int, int>>> _messageMatches = {};
+  final List<int> _messageIndices = [];
```

Comment:
I don't think that these should be 'final'
@Bishoywadea

Reply:
In the current logic, these variables are not being set in the code but in the future, they should be changed.